src/main/kotlin/com/exactpro/th2/conn/utility/EventStoreExtensions.kt

```diff
@@ -35,36 +34,66 @@ import mu.KotlinLogging
 
 private val LOGGER = KotlinLogging.logger { }
 
-@JvmOverloads
 @Throws(JsonProcessingException::class)
 fun MessageRouter<EventBatch>.storeEvent(
     event: Event,
-    parentEventId: EventID? = null
-): Event {
+    parentEventId: EventID,
+) = storeEvent(event, parentEventId.bookName)
```

Comment:
Why do we use parentEventId.bookName only instead of full parent event id?

Reply:
Done, I confused myself. Thanks!